stdlib/in_channel.mli

```diff
@@ -150,3 +150,9 @@ val set_binary_mode : t -> bool -> unit
 
     This function has no effect under operating systems that do not distinguish
     between text mode and binary mode. *)
+
+val isatty : t -> bool
+(** [isatty ic] is [true] if [ic] refers to a terminal or console window,
+    [false] otherwise.
```

Comment:
A nitpick is that the [Windows documentation for isatty](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/isatty?view=msvc-170) refers to:
>  a character device (a terminal, console, printer, or serial port).

Perhaps it's more accurate to refer to a character device instead, but I haven't checked to see what isatty on Linux does does if you give it a serial port device.

Reply:
> I haven't checked to see what isatty on Linux does does if you give it a serial port device.

It returns `false`. Personally I would keep the existing somewhat vague docstring.